### DISCUSSION TITLE
Opensearch dashboard is not coming up

### DISCUSSION THREAD

**Discussion by bommasv** [Opster/opensearch-k8s-operator#797] [Q&A] [answered]
I am following the documentation and I have setup the opensearch operator flow with operator version 2.6.0. When I grep the pod logs of  opensearch operator it says 

http: TLS handshake error from <IP-Address>, the same IP address the operator has connected to. 

After 5 minutes, I did helm install for cluster (OpenSearchCluster), upon the deploy the security config pod (opensearch-securityconfig-update-1234j) is created when I checked the logs it says 

"Waiting to connect to the cluster", apart from these I don't see any logs, and I am using 2.3.0 version. 

After 1 day the operator is getting crash looped because the liveness, redness probes are getting failed. 

The security pod is no more visible in pods.  Can you suggest me where I am going wrong. The code is 70 match to the code exists in main branch.

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @bommasv.

> http: TLS handshake error from

That is from a sidecar of the operator pod and not a real error. You'll need to check the logs from the operator-controller-manager container (`kubectl logs <pod> -c operator-controller-manager`)

> "Waiting to connect to the cluster",

That is also normal as it takes a few minutes for the cluster to get into a state where the job can apply the securityconfig.

You'll need to check the actual opensearch cluster pods if they are starting. If not you might want to check the underlying statefulsets if there is a problem. And also check the operator logs if it reports any errors.

**Comment by bommasv** [comment]
Thank You @swoehrl-mw
